src/SSHDebugPS/UI/ViewModels/ContainerPickerViewModel.cs

```diff
@@ -162,6 +162,26 @@ private void RefreshContainersListInternal()
                     containers = DockerHelper.GetRemoteDockerContainers(connection, Hostname, out totalContainers);
                 }
 
+                foreach(DockerContainerInstance container in containers)
+                {
+                    try
+                    {
+                        container.Platform = DockerHelper.GetContainerPlatform(Hostname, container.Name, false);
+                        try
```

Comment:
please remove the inner tryblock. That is not necessary. Also, why is this a blanket catch statement?